docs/tests/integration/post_v1_message-batches/happy_path.md

```diff
@@ -18,6 +18,21 @@
 - Response contains correctly formatted link to new message URI
 
 
+### Scenario: An API consumer creating a batch of messages with valid contact details receives a 201 response
+
+**Given** the API consumer provides valid contact details for a recipient in their new message batch
+<br/>
+**When** the request is submitted
+<br/>
+**Then** the response is a 201 success
+<br/>
+
+**Asserts**
+- Response returns a 201 status code
+- Response body matches expected result
+- Response contains correctly formatted link to new message URI
```

Comment:
Is that correct for a batch response? I thought we only returned the request ID at the moment.

Reply:
The documentation is automatically generated, I'm not sure how it came up with that. What do you think it should be instead?

Reply:
I think it's coming from `test_201_message_batch_valid_contact_details.rst`. Maybe it could be changed to
`Response contains request ID`

I'm not entirely sure what his documentation is for to be honest :D